docker-compose.yml

```diff
@@ -45,6 +49,7 @@ services:
       - dev-db
       - test-db  # use -e SQLALCHEMY_TEST_DATABASE_URI=... to exec pytest
       - queue-db
+      - mailhog  # Added MailHog dependency
```

Comment:
This comment is redundant, we are in a section named `depends_on` :)

Reply:
okay right i'll remove it